<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeteoECI_GUI</class>
 <widget class="QMainWindow" name="MeteoECI_GUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>382</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>382</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>382</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MeteoECI_GUI</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/artificial-intelligencia.png</normaloff>:/icons/artificial-intelligencia.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>197</width>
      <height>73</height>
     </rect>
    </property>
    <property name="title">
     <string>Horas - Minutos</string>
    </property>
    <widget class="QLineEdit" name="txt_Hr">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>34</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_Min">
     <property name="geometry">
      <rect>
       <x>102</x>
       <y>34</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>368</width>
      <height>73</height>
     </rect>
    </property>
    <property name="title">
     <string>GPS</string>
    </property>
    <widget class="QLineEdit" name="txt_Lat">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>34</y>
       <width>108</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_Lon">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>34</y>
       <width>108</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_Alt">
     <property name="geometry">
      <rect>
       <x>245</x>
       <y>34</y>
       <width>108</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>253</width>
      <height>73</height>
     </rect>
    </property>
    <property name="title">
     <string>Temperatura</string>
    </property>
    <widget class="QLineEdit" name="txt_Hum">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>34</y>
       <width>108</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_Temp">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>34</y>
       <width>108</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>280</y>
      <width>169</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="Back">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Next">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>253</width>
      <height>73</height>
     </rect>
    </property>
    <property name="title">
     <string>Viento</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>34</y>
       <width>225</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="txt_Vel"/>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_Dir"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>120</y>
      <width>136</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Precipitaci√≥n</string>
    </property>
    <widget class="QLineEdit" name="txt_Preci">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>33</y>
       <width>108</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>160</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Estado:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="estado"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
